crates/aide/src/axum/mod.rs

```diff
@@ -495,7 +543,9 @@ where
         path: &str,
         method_router: impl Into<ApiMethodRouter<S>>,
     ) -> Self {
-        self.router = self.router.route(path, method_router.into().router);
+        self.router = self
+            .router
+            .route_with_tsr(path, method_router.into().router);
```

Comment:
This looks like an unrelated bugfix, can you separate it into another PR, or add a changelog entry?

Reply:
Will add a changelog entry